packages/@react-spectrum/badge/docs/Badge.mdx

```diff
@@ -51,13 +51,19 @@ import {Text} from '@react-spectrum/text';
 import CheckmarkCircle from '@spectrum-icons/workflow/CheckmarkCircle';
 
 <Badge variant="positive">
-  <CheckmarkCircle />
+  <CheckmarkCircle aria-label="Done" />
   <Text>Icon + Label</Text>
 </Badge>
 ```
 
 ### Accessibility
-If a visible label isn't specified, an `aria-label` must be provided for accessibility.
+If a visible label isn't specified, an `aria-label` must be provided to the icon for accessibility.
+
+```tsx example
```

Comment:
glad you made changes, makes sense it goes on the icon not the badge itself now that I see it. Given that the aria-label should be on every icon as your example above suggests, I think this example shouldn't be here, especially since we want to discourage icon only. Instead we should just say that an aria-label must be provided to an icon in a badge.